src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -56,21 +56,45 @@ class IfTypes(int, Enum):
 class ArpUpdater(MIBUpdater):
     def __init__(self):
         super().__init__()
+        self.db_conn = Namespace.init_namespace_dbs()
         self.arp_dest_map = {}
         self.arp_dest_list = []
         self.arp_dest_map = {}
         self.arp_dest_list = []
+        self.neigh_key_list = {}
 
-    def update_data(self):
-        self.arp_dest_map = {}
-        self.arp_dest_list = []
+    def reinit_data(self):
+        if len(self.db_conn) > 1:
+            Namespace.connect_all_dbs(self.db_conn, mibs.APPL_DB)
```

Comment:
>connect_all_dbs [](start = 22, length = 15)

if only one db, will you connect?

Reply:
Yes, changed to make sure we connect always.